apps/src/templates/sectionAssessments/FreeResponsesAssessments.jsx

```diff
@@ -0,0 +1,125 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+
+export const COLUMNS = {
+  STUDENT: 0,
+  RESPONSE: 1,
+};
+
+const freeResponsesDataPropType = PropTypes.shape({
+  id:  PropTypes.number,
+  studentId: PropTypes.string,
+  name: PropTypes.string,
+  response: PropTypes.string,
+});
+
+class FreeResponsesAssessments extends Component {
+  static propTypes= {
+    freeResponses:PropTypes.arrayOf(freeResponsesDataPropType),
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc',
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  studentResponseColumnFormatter = (studentAnswers, {rowData, rowIndex, columnIndex}) => {
```

Comment:
you can remove `rowData` and `columnIndex` as parameters if you aren't using them. here, you're [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) the object that reactabular passes as its second argument, so unlike normal method parameters where it relies on the ordering of params, you can grab the keys you need from the object by name. a common example you'll see is this:
```
const {firstName, lastName} = this.props.student;
```
where `this.props.student` is an object with a bunch of keys, but you only care about pulling out `firstName` and `lastName`

Reply:
This is a very neat feature and I have tried to implement it and I am not getting the desired outcome.

Reply:
so you're saying when you remove `rowData` or `columnIndex` (since they are unused) that accessing `rowIndex` no longer works?

Reply:
I also removed rowData and columnIndex.